pkg/backend/kubeflow/generator/cm.go

```diff
@@ -0,0 +1,213 @@
+package generator
+
+import (
+	"fmt"
+
+	s2iconfigmap "github.com/caicloud/ciao/pkg/s2i/configmap"
+	pytorchv1alpha2 "github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1alpha2"
+	tfv1alpha2 "github.com/kubeflow/tf-operator/pkg/apis/tensorflow/v1alpha2"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/caicloud/ciao/pkg/types"
+)
+
+const (
+	baseImageTF      = "tensorflow/tensorflow:1.10.1-py3"
+	baseImagePyTorch = "pytorch/pytorch:v0.2"
+)
+
+// CM is the type for CM generator.
+type CM struct {
+}
+
+// NewCM returns a new CM generator.
+func NewCM() *CM {
+	return &CM{}
+}
+
+// GenerateTFJob generates a new TFJob.
+func (c CM) GenerateTFJob(parameter *types.Parameter) *tfv1alpha2.TFJob {
+	psCount := int32(parameter.PSCount)
+	workerCount := int32(parameter.WorkerCount)
+
+	mountPath := fmt.Sprintf("/%s", parameter.Image)
+	filename := fmt.Sprintf("/%s/%s", parameter.Image, s2iconfigmap.FileName)
+
+	return &tfv1alpha2.TFJob{
+		TypeMeta: metav1.TypeMeta{
+			Kind: tfv1alpha2.Kind,
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      parameter.GenerateName,
+			Namespace: metav1.NamespaceDefault,
```

Comment:
can I use other namespaces?

Reply:
Sure, but I prefer to open a new PR to support it. Maybe in the config file to customize the namespace. I will open an issue.